Framework/CurveFitting/src/Algorithms/Fit.cpp

```diff
@@ -212,8 +209,16 @@ void Fit::finalizeMinimizer(size_t nIterations) {
                    std::to_string(m_maxIterations) + " iterations.";
   }
 
+  if (errorString.empty()) {
+    errorString = "success";
+  }
+
   // return the status flag
   setPropertyValue("OutputStatus", errorString);
+  setProperty("OutputNIterations", static_cast<int>(nIterations));
+  if (!this->isChild()) {
+    g_log.notice(errorString);
```

Comment:
I wonder, if it would make sense to be a bit more wordy for the log, for example write
```
Fit status: success
```
In case of `errorString != "success"`, the error message could be written to `g_log.error()` instead of `g_log.notice()`.